cmake/SetupCompilerOptions.cmake

```diff
@@ -121,12 +121,18 @@ message(STATUS "OpenMP Support is ${ENABLE_OPENMP}")
 if(ENABLE_OPENMP)
     find_package(OpenMP REQUIRED)
     message(STATUS "OpenMP CXX Flags: ${OpenMP_CXX_FLAGS}")
-    
+
+    if (ENABLE_FORTRAN AND NOT OpenMP_CXX_FLAGS STREQUAL OpenMP_Fortran_FLAGS)
```

Comment:
Perhaps it would be a good idea to add a line about why this is here/necessary?

Reply:
Yep, added a comment.